src/tests/bounding_box_of cylinder.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
I suggest you change the filename to `bounding_box_test_input` and put the inputs related to bounding box for all shapes there.